torch/quantization/observer.py

```diff
@@ -567,6 +567,7 @@ def __init__(self, bins=2048, dtype=torch.quint8,
         self.min_val = None
         self.max_val = None
         self.dst_nbins = 2 ** torch.iinfo(self.dtype).bits
+        self.upsample_rate = 128
```

Comment:
May add comment on why choose 128 here? And could make upsample_rate as an input argument.